packages/@coorpacademy-components/src/molecule/setup-cohort-item/test/fixtures/create-new.js

```diff
@@ -0,0 +1,102 @@
+export default {
+  props: {
+    fields: [
+      {
+        type: 'text',
+        title: 'Name',
+        value: '',
+        required: true,
+        autoFocus: true
+      },
+      {
+        type: 'splitScreen',
+        conditions: {
+          title: 'The Provider',
+          values: [
+            {
+              type: 'checkbox',
+              title: 'Provider 1',
+              checked: false,
+              value: 'them_cascsa'
+            },
+            {
+              type: 'checkbox',
+              title: 'Provider 2',
+              checked: false,
+              value: 'them_3r2fwc'
+            },
+            {
+              type: 'checkbox',
+              title: 'Provider 3',
+              checked: false,
+              value: 'them_wefewgweb'
+            },
+            {
+              type: 'checkbox',
+              title: 'Provider 4',
+              checked: false,
+              value: 'them_bwegwegwe'
+            },
+            {
+              type: 'checkbox',
+              title: 'Provider 5',
+              checked: false,
+              value: 'them_gwegwevfwe'
+            },
+            {
+              type: 'checkbox',
+              title: 'Provider 6',
+              checked: false,
+              value: 'them_ewgwebh'
+            }
+          ]
+        },
+        groups: {
```

Comment:
I would expect a group array. The name is not good. I propose `groupSelection`

Reply:
It's Done